v7/libretto/assets/xml/rss.xsl

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not inheriting from base for this theme. It's a 'from scratch' implementation. On my browser at least, things work better with this file included.